src/main/java/codesquad/springcafe/controller/UserController.java

```diff
@@ -0,0 +1,47 @@
+package codesquad.springcafe.controller;
+
+import codesquad.springcafe.model.User;
+import codesquad.springcafe.service.UserService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/users")
+public class UserController {
+
+    private final UserService userService;
+
+    @Autowired
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping
+    public String signUp(@ModelAttribute("user") User user, Model model) {
+        User newUser = userService.createUser(user.getUserId(), user.getPassword(), user.getName(), user.getEmail());
```

Comment:
로직은 이해가 되는데 UserDTO와 newUser 사이에 어떤 차이가 있는지 잘 모르겠어요. 
없어도 되는 코드로 보입니다.
```suggestion
        User newUser = userService.createUser(userDTO);
```